spec/classes/mod/php_spec.rb

```diff
@@ -290,12 +279,9 @@
       end
 
       # all the following tests are for legacy php/apache versions. They don't work on modern ubuntu and redhat 8
-      next if (facts[:os]['release']['major'].to_i >= 15 && facts[:os]['name'] == 'SLES')  ||
-              (facts[:os]['family'] == 'Debian')                                           ||
-              (facts[:os]['release']['major'].to_i >= 8 && (facts[:os]['name'] == 'RedHat' || facts[:os]['name'] == 'CentOS' ||
-                                                            facts[:os]['name'] == 'Rocky'  || facts[:os]['name'] == 'AlmaLinux'))
-
-      describe 'OS independent tests' do
+      skip_os = ['Debian', 'RedHat']
+      describe 'OS independent tests',
+               unless: skip_os.include?(facts[:os]['family'] && facts[:os]['release']['major'].to_i > 7) || (facts[:os]['release']['major'].to_i >= 15 && facts[:os]['name'] == 'SLES') do
```

Comment:
That's an interesting hack. Technically you're right that Debian > 7 should be skipped. Though I always find `>= 8` easier to read than `> 7`. Is that just me?